fe/fe-core/src/test/java/org/apache/doris/nereids/postprocess/RuntimeFilterTest.java

```diff
@@ -0,0 +1,186 @@
+                );
+    }
+
+    @Test
+    public void testNestedJoinGenerateRuntimeFilter() throws AnalysisException {
+        String sql = SSBUtils.Q4_1;
+        PlanChecker.from(connectContext)
+                .analyze(sql)
+                .implement()
+                .postProcess()
+                .matchesPhysicalPlan(
+                        physicalQuickSort(
+                                physicalAggregate(
+                                        physicalAggregate(
+                                                physicalHashJoin(
+                                                        physicalHashJoin(
+                                                                physicalHashJoin(
+                                                                        physicalHashJoin(
+                                                                                physicalOlapScan(),
+                                                                                physicalOlapScan()
+                                                                        ),
+                                                                        physicalFilter(
+                                                                                physicalOlapScan()
+                                                                        )
+                                                                ),
+                                                                physicalFilter(
+                                                                        physicalOlapScan()
+                                                                )
+                                                        ),
+                                                        physicalFilter(
+                                                                physicalOlapScan()
+                                                        )
+                                                ).when(join -> {
+                                                    List<RuntimeFilter> filters = join.getRuntimeFilters().getNereridsRuntimeFilter();
+                                                    return filters.size() == 4
+                                                            && checkRuntimeFilterExpr(filters.get(0), "p_partkey", "lo_partkey")
+                                                            && checkRuntimeFilterExpr(filters.get(1), "s_suppkey", "lo_suppkey")
+                                                            && checkRuntimeFilterExpr(filters.get(2), "c_custkey", "lo_custkey")
+                                                            && checkRuntimeFilterExpr(filters.get(3), "lo_orderdate", "d_datekey");
+                                                })
+                                        )
+                                )
+                        )
+                );
+    }
+
+    @Test
+    public void testTranslateSSB() throws Exception {
```

Comment:
this UT is not good since we cannot ensure runtime filter work well through this UT